Specs/DateTime/English/DateTimeModelCalendarMode.json

```diff
@@ -898,42 +898,5 @@
         }
       }
     ]
-  },
-  {
-    "Input": "Mortgages were at 6.45 percent",
```

Comment:
Why delete these specs?

Reply:
These should be in the ComplexCalendar spec.